src/semantic-ui/site/globals/site.overrides

```diff
@@ -137,6 +137,11 @@ a {
   justify-content: center;
 }
 
+.pagination [type=ellipsisItem]:nth-last-child(3) + [type=pageItem] {
```

Comment:
To double check if it makes sense and test it:
```suggestion
.ui.pagination [type=ellipsisItem]:nth-last-child(3) + [type=pageItem] {
```
just because I saw that there are 2 paginations cmps and classes:
```
<div class="pagination">
    <div class="ui pagination menu" ...>
```

Reply:
Thanks, it's indeed safer to specify the `ui` class.